packages/@aws-cdk/aws-neptune-alpha/test/cluster.test.ts

```diff
@@ -879,6 +879,40 @@ describe('DatabaseCluster', () => {
       RetentionInDays: 30,
     });
   });
+
+  test('cluster with copyTagsToSnapshot default', () => {
```

Comment:
How about the description?

```suggestion
  test('copyTagsToSnapshot is not set by default', () => {
```